cvat/apps/quality_control/models.py

```diff
@@ -205,6 +205,8 @@ class QualitySettings(models.Model):
 
     low_overlap_threshold = models.FloatField()
 
+    use_bbox_size_for_points = models.BooleanField()
```

Comment:
Have you considered using an enum for this? E.g. call it `points_size_method` and have the values be `bbox_size` and `image_size`. Neither of these methods seems particularly "true" or "false" to me, so perhaps an enum would be more understandable?

Reply:
It's hard to say currently if there are going to be more methods in future. Basically, you can represent any boolean as a enum. I think currently it's premature.

Reply:
> It's hard to say currently if there are going to be more methods in future.

I didn't say anything about more methods. I'm saying that the semantics don't seem boolean-like here.

> Basically, you can represent any boolean as a enum.

Certainly, but sometimes a boolean makes the meaning clear, and sometimes not. E.g., for `compare_groups` a boolean makes sense: if it's true, you compare groups, and if it's false, you don't.

Here you do something in either case, so why should the "bbox size" method be true and "image size" be false? It seems arbitrary. Furthermore, it's impossible to determine what `use_bbox_size_for_points=False` means without consulting the documentation.

Reply:
What do you think of `point_size_base` or `point_distance_base` for the enum name? The members are `image_size`, `group_bbox_size`.

Reply:
Updated to `point_size_base`. This determines the scale factor for the relative radius (configured in `oks_sigma`) of each point in the points annotation or in a group of such.